modules/030-cloud-provider-dvp/images/cloud-controller-manager/werf.inc.yaml

```diff
@@ -0,0 +1,54 @@
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}
+fromImage: common/distroless
+import:
+  - artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+    add: /dvp-cloud-controller-manager
+    to: /dvp-cloud-controller-manager
+    before: setup
+docker:
```

Comment:
plaese use imageSpecs, for example
https://github.com/deckhouse/deckhouse/blob/c5dc9cb0606599fb3d6d99c46c09b7ece4a42b42/ee/be/modules/140-user-authz/images/webhook/werf.inc.yaml#L13-L12